test/interfacer_tests.jl

```diff
@@ -85,3 +92,68 @@ end
     @test parent(get_field(stub, Val(:area_fraction)))[1] == FT(1)
     @test parent(get_field(stub, Val(:surface_temperature)))[1] == FT(2)
 end
+
```

Comment:
Instead of having tests, we could always do asserts in the init functions. What do you think?

Reply:
I'm not sure what we would assert. I was originally thinking of writing a function to check if a field is continuous, then assert that that function is true for each initialized field. But since it's nontrivial to check if a field is continuous, I don't know what we should assert if we take that approach

Reply:
I think it makes sense to move the functions to the init files, but it would be nice to keep the tests for them too. I wonder if we should have a place to add tests for component model functions too. That might make it easier for the component model developers when they have to move these functions to their own repos

Reply:
I see what you mean. We already have [component_model_tests.jl](https://github.com/CliMA/ClimaCoupler.jl/blob/main/test/component_model_tests.jl), which will be converted to a folder with the individual components in the sea ice PR. We can add it here.